home.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home page</title>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css" rel="stylesheet">
+    <script src="script.js" defer></script>
+</head>
+<body>
+
+    <!-- First Navbar -->
+    <nav class="navbar social-navbar">
+        <ul>
+            <li><a href="https://twitter.com" target="_blank" rel="noopener"><i class="fab fa-twitter"></i></a></li>
+            <li><a href="https://facebook.com" target="_blank" rel="noopener"><i class="fab fa-facebook"></i></a></li>
+            <li><a href="English">English</a></li>
+            <li><a href="mypage">Mypage</a></li>
+            <li><a href="Logout">Logout</a></li>
+        </ul>
+    </nav>
+
+    <!-- Second Navbar with Hamburger Menu -->
+<nav class="navbar main-navbar">
+    <div class="logo">
+        <a href="about.html">Global<b> Innovation</b></a>
+    </div>
+
+    <!-- Hamburger Icon -->
+    <div class="hamburger" id="hamburger">
+        <i class="fas fa-bars"></i>
+    </div>
+
+    <!-- Nav Links -->
+    <div class="nav-links-container" id="nav-links-container">
+        <ul class="nav-links" id="nav-links">
+            <li><a href="about.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="about.html">Sponsors</a></li>
+            <li><a href="#">Speakers</a></li>
+            <button id="ctaButton" class="cta-button" type="button">Innovation Schedule</button>
+        </ul>
+    </div>
+</nav>
+
+
+    <!-- Content -->
+    <div class="container">
+        <h3>"Hello! Sharing World"</h3>
+        <h1>Global Innovation<br> Exchange Summit<br>2024</h1>
+        <p class="description">
+            <h6>A prestigious, forward-thinking event that brings together global leaders,<br> entrepreneurs, innovators, and investors to explore and showcase the latest<br> advancements in technology, sustainability, and creative industries</h6>
+        </p>
+        <p class="event-date"><h4>2024-11-15(THU) ~ 18(FRI)</h4></p>
+        <p class="location"><h5>@Javits Center 429 11th Ave, New York, NY 10001, United States</h5></p>
+    </div>
+
```

Comment:
@Aqsamaan12 Great job so far, but you home page content is not completed, you are displaying the home page content from your menu but it should be present on the main screen, along with this header content. Like one long page for the main screen which will include header, `Main Program`, feature speakers and the footer on the same page.
Please fix that!!